libraries/AP_Quicktune/AP_Quicktune.h

```diff
@@ -0,0 +1,184 @@
+#pragma once
+
+
+#include <AP_Vehicle/AP_Vehicle_Type.h>
+#include <AP_HAL/AP_HAL_Boards.h>
+#ifndef QUICKTUNE_ENABLED
+ #define QUICKTUNE_ENABLED 1 //APM_BUILD_TYPE(APM_BUILD_ArduCopter)
```

Comment:
One of the main points of doing this was so that 1MB boards (that don't have scripting) can also use this so that would quite defeat the purpose.

The other is so that we can put this in the wiki rather than just going "tune manually before trying autotune", which results in people going "I dunno how to do that" and just going directly going to autotune, at which point it probably crashes their copter. Can't do that if it's not in the boards that the majority of people are probably still gonna buy.

Reply:
@andyp1per it is now off by default for all copters, enable in custom build server